pkgs/by-name/nc/nchat/package.nix

```diff
@@ -56,7 +100,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DCMAKE_INSTALL_LIBDIR=lib"
-    "-DHAS_WHATSAPP=OFF" # go module build required
+    "-DHAS_WHATSAPP=ON" # go module build required
```

Comment:
The default is already on, so I think this line should be removed: https://github.com/d99kris/nchat/blob/master/CMakeLists.txt#L51